apps/web-client/src/lib/stores/inventory/table_content.ts

```diff
@@ -3,43 +3,81 @@ import { debug } from '@librocco/shared';
 
 import type { NoteInterface, WarehouseInterface } from '@librocco/db';
 
-import type { BookStore, DisplayRow, PaginationData } from '$lib/types/inventory';
+import type { PaginationData, FullTableRow } from '$lib/types/inventory';
 
 import { readableFromStream } from '$lib/utils/streams';
+import { from, map, Observable, switchMap, tap } from 'rxjs';
+import { db } from '$lib/db';
+import type { DebugCtx } from '@librocco/shared/dist/debugger';
 
 interface CreateDisplayEntriesStore {
 	(
 		entity: NoteInterface | WarehouseInterface | undefined,
 		currentPageStore: Readable<number>,
-		bookStore: Readable<BookStore>,
 		ctx: debug.DebugCtx
-	): Readable<DisplayRow[]>;
+	): Readable<FullTableRow[]>;
 }
+
+interface createDisplayRowStream {
+	(entity: NoteInterface<object> | WarehouseInterface<NoteInterface<object>, object>, ctx: DebugCtx): Observable<
+		FullTableRow[]
+	>;
+}
+
+const createDisplayRowStream: createDisplayRowStream = (entity, ctx) => {
+	const fullTableRow = entity?.stream(ctx).entries.pipe(
+		switchMap((valueFromEntryStream) => {
+			const isbns = valueFromEntryStream.map((entry) => entry.isbn);
+			// map entry to just isbns
```

Comment:
I gave you leads on why you did't receive anything (in the test) from `entity?.stream(ctx).entries` (note wasn't updated properly). For further checks, you can add `.tap` above the `switchMap` here